controllers/controller.js

```diff
@@ -42,3 +43,20 @@ exports.getCommentsByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  const newReview = {
+    body: req.body.body,
+    votes: 0,
+    author: req.body.username,
+    review_id: review_id,
+    created_at: new Date(1610964545411),}
```

Comment:
You don't want to hardcode the date of creation for a new comment. Fortunately, you don't have to set it at all - if you look at the database schema setup in the seed file, you will see that the `votes` columns actually has a default value, so you don't need to include it as part of the `INSERT` statement at all.